src/libraries/Directory.Build.props

```diff
@@ -66,6 +68,13 @@
     <NoWarn>$(NoWarn);SYSLIB0011</NoWarn>
   </PropertyGroup>
 
+  <!-- Project references shouldn't be copied to the output for non test apps. -->
+  <ItemDefinitionGroup Condition="'$(IsTestProject)' != 'true' and '$(IsTestSupportProject)' != 'true'">
+    <ProjectReference>
```

Comment:
Would it make sense to put this next to `UpdateProjectReferencesWithPrivateAttribute`?